ivadomed/main.py

```diff
@@ -274,12 +281,15 @@ def cmd_train(context):
                                                                                            "gamma"],
                                                                                        context["loss"]["params"][
                                                                                            "alpha"]))
+        elif context["loss"]["name"] == "multi_class_dice":
+            loss_fct = losses.MultiClassDiceLoss(classes_of_interest=context["loss"]["params"]["classes_of_interest"])
 
         if not context["loss"]["name"].startswith("focal"):
             print("\nLoss function: {}.\n".format(context["loss"]["name"]))
 
     else:
-        print("Unknown Loss function, please choose between 'dice', 'focal', 'focal_dice', 'gdl' or 'cross_entropy'")
+        print("Unknown Loss function, please choose between 'dice', 'focal', 'focal_dice', 'gdl', 'cross_entropy',"
+              "'multi_class_dice' or 'combined_dice'")
```

Comment:
'combined dice' does not appear in the option line 260, and I don't see it in the previous `if...elif` statement. Unless this is implemented in another loss function (multiclass) ?

Reply:
Yes forgot to remove it, good catch thanks :)